src/interfaces/IRouter.sol

```diff
@@ -133,16 +133,27 @@ interface IRouter {
     function rebalanceSolver(address solver, bytes32 requestId, bytes calldata signature) external;
 
     /// @notice Relays tokens to the recipient and stores a receipt
-    /// @param token The token being relayed
+    /// @param requestId The original request ID from the source chain
+    /// @param sender The sender of the swap request on the source chain
     /// @param recipient The target recipient of the tokens
+    /// @param tokenIn The token being relayed on the destination chain
```

Comment:
this comment is slightly confusing to me - shouldn't it be "the token deposited on the source chain"?